CMakeLists.txt

```diff
@@ -224,8 +226,8 @@ if ( KWIVER_ENABLE_MATLAB )
   file( APPEND "${KWIVER_SETUP_SCRIPT_FILE}" "export LD_LIBRARY_PATH=${Matlab_LIBRARY_DIR}:$LD_LIBRARY_PATH\n" )
 endif()
 
-if ( fletch_ROOT )
-  file( APPEND "${KWIVER_SETUP_SCRIPT_FILE}" "export LD_LIBRARY_PATH=${fletch_ROOT}/lib:$LD_LIBRARY_PATH\n" )
+if ( fletch_FOUND )
+  file( APPEND "${KWIVER_SETUP_SCRIPT_FILE}" "export LD_LIBRARY_PATH=${fletch_DIR}/lib:$LD_LIBRARY_PATH\n" )
```

Comment:
That is no longer true.  If you update and rebuild your Fletch you'll see that fletch_ROOT no longer exists.  fletchConfig.cmake has been moved into the install dir so now fletch_DIR must point to the install dir instead of the build dir and it serves both purposes.

I think that change in Fletch makes sense, but it would have been better if it was done more gradually to prevent things from breaking.  Oh well.